src/XRP/xrp-currency.ts

```diff
@@ -2,19 +2,15 @@ import { Currency } from '../generated/web/org/xrpl/rpc/v1/amount_pb'
 
 /**
  * An issued currency on the XRP LEdger
- * - SeeAlso: https://xrpl.org/currency-formats.html#currency-codes
+ * @see: https://xrpl.org/currency-formats.html#currency-codes
  */
 export default class XRPCurrency {
-  public name: string
-
-  public code: string | Uint8Array
-
   public static from(currency: Currency): XRPCurrency | undefined {
-    return new XRPCurrency(currency)
+    return new XRPCurrency(currency.getName(), currency.getCode())
   }
 
-  private constructor(currency: Currency) {
-    this.name = currency.getName()
-    this.code = currency.getCode()
-  }
+  private constructor(
+    readonly name: string,
+    readonly code: string | Uint8Array,
```

Comment:
We should document this type. If it's a string, then it's a base64 encoded string. If it's an array, it's an array of bytes. 

You can use a more specific helper:
```
# Gets string | Uint8Array
currency.getCode() 

# Gets a uint8array
currency.getCode_asU8()

# Gets a base64 encoded string
currency.getCode_asB64()
```

I found this by inspecting the generated code on disk: 

<img width="1432" alt="Screen Shot 2020-03-16 at 4 30 44 PM" src="https://user-images.githubusercontent.com/18271723/76763074-7d6f5580-67a3-11ea-9e1a-d4b97aa92afc.png">

Reply:
great, took care of this on (2/n) and merged here